readthedocs/proxito/urls.py

```diff
@@ -86,6 +88,21 @@
     #     name='docs_detail',
     # ),
 
+    # External versions
+    # (RTD_EXTERNAL_VERSION_DOMAIN/html/<project-slug>/<version-slug>/<filename>)
+    # NOTE: requires to be before single version
+    url(
+        (
+            r'^external/html/(?P<project_slug>{project_slug})/'
```

Comment:
Since we are serving in a complete different domain that already includes the `external` word, we may be able to remove the `/external/` part from this URL.

This will involve adapting [`Project.get_storage_path`](https://github.com/readthedocs/readthedocs.org/blob/0b41a1fcaf3c00ee0a89bbd3b48abd594f73aa4d/readthedocs/projects/models.py#L559-L562) and creating a new rule for NGINX that appends `/external` to the `proxy_pass` value when serving this domain.

We don't have to do this in PR, though.

Reply:
I adapted the code to avoid the `external/` in the user-facing URL but keep it when serving it from El Proxito.